packages/shylu/shylu_dd/frosch/src/CoarseSpaces/FROSch_CoarseSpace_def.hpp

```diff
@@ -51,77 +51,127 @@ namespace FROSch {
     using namespace Xpetra;
 
     template <class SC,class LO,class GO,class NO>
-    CoarseSpace<SC,LO,GO,NO>::CoarseSpace() :
-    SerialRowMap_ (),
+    CoarseSpace<SC,LO,GO,NO>::CoarseSpace(CommPtr mpiComm,
+                                          CommPtr serialComm) :
+    MpiComm_ (mpiComm),
+    SerialComm_ (serialComm),
     UnassembledBasesMaps_ (0),
```

Comment:
Prefer member initializers.  That way, you won't have to set all those integers to zero explicitly in each constructor.

Reply:
Actually, I prefer it like this. Then, I know exactly what happens.

Reply:
@searhein Preferred form for C++11 code is to use member initializers.  This reduces duplicate code in constructors, and avoids bugs like forgetting to initialize when you add a new member or a new constructor.